tools/validators/instance_validator/tests/fake_instances/GOOD/translation_units.yaml

```diff
@@ -17,7 +17,7 @@
 CHWS_WDT-17-GUID:
   type: HVAC/CHWS_WDT
   code: CHWS_WDT-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980753"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.